server/lib/chat/chat-models.ts

```diff
@@ -265,6 +265,21 @@ export async function getMessagesForChannel(
   }
 }
 
+export async function deleteChannelMessage(
+  messageId: string,
+  withClient?: DbClient,
+): Promise<void> {
+  const { client, done } = await db(withClient)
+  try {
+    await client.query(sql`
+      DELETE FROM channel_messages
+      WHERE id = ${messageId};
```

Comment:
Doing it this way doesn't validate at all that the message is in the channel it was requested through? I get that admins can delete anything anywhere anyway and the message ID is unique on its own, but you 100% are just going to reuse this model method for the non-admin version and it's going to be a problem